designs/lily/src/waistband.mjs

```diff
@@ -0,0 +1,121 @@
+import { back } from './back.mjs'
+import { front } from './front.mjs'
+// do not import from Paco since that would use Paco's back to define the waistband
+// import { waistband as pacoWaistband } from '@freesewing/paco'
+
+function lilyWaistband({
+  utils,
```

Comment:
```suggestion
```
lint error. Utils is not used,

Reply:
(this entire file has been replaced now)